services/storage-users/pkg/command/trash_bin.go

```diff
@@ -0,0 +1,89 @@
+package command
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cs3org/reva/v2/pkg/events"
+	"github.com/owncloud/ocis/v2/ocis-pkg/config/configlog"
+	"github.com/owncloud/ocis/v2/services/storage-users/pkg/config"
+	"github.com/owncloud/ocis/v2/services/storage-users/pkg/config/parser"
+	"github.com/owncloud/ocis/v2/services/storage-users/pkg/event"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	flagUserID = &cli.StringFlag{
+		Name:  "user-id",
+		Usage: "The user-id of the user who should be used to list and delete expired space trash-bin items",
```

Comment:
will there be a change when the sevice user is implemented?

Reply:
yes, but then we change the command to use the service user as default instead of the admin user.